plugins/lsp/language-servers/default.nix

```
@@ -443,6 +443,12 @@ let
       name = "pyright";
       description = "pyright for Python";
     }
+    {
+      name = "r-language-server";
+      description = "Enable languageserver LSP for R";
```

```suggestion
      description = "languageserver for R";
```

I think this is more consistent with other descriptions?